web-ui/src/main/resources/catalog/components/edit/onlinesrc/OnlineSrcService.js

```diff
@@ -366,6 +366,7 @@
             scopedName: qParams.name,
             uuidref: qParams.uuidDS,
             uuid: qParams.uuidSrv,
+            source: qParams.identifier ? qParams.identifier : '',
```

Comment:
`qParams.identifier || ''`

Reply:
Changed all the ternary `?` operator by lazy evaluation.